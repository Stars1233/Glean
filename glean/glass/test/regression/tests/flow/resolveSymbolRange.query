action: symbolLocation
args: "test/js/imports/C"
