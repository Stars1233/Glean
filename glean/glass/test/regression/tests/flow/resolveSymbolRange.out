[
    "@generated",
    {
        "filepath": "test/imports.js",
        "range": {
            "columnBegin": 21,
            "columnEnd": 22,
            "lineBegin": 32,
            "lineEnd": 32
        },
        "repository": "test"
    }
]